data/benchmarking/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Ignore everything in this directory
```

Comment:
Don't have a gitignore here, just use the top level one

Reply:
Done